test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -1177,29 +1167,8 @@ describe("POST tests", () => {
     expect(resp.text).toEqual(`${FOUND_REDIRECT_TO} ${PAYMENT_LINK_JOURNEY}`);
   });
 
-  test(`redirect the ${CONFIRMATION_PAGE} page after a successful post from ${CHECK_YOUR_ANSWERS_PAGE} page`, async () => {
-    mockIsActiveFeature.mockReturnValueOnce( false ); // For Save and Resume
-    const resp = await request(app).post(CHECK_YOUR_ANSWERS_URL);
-
-    expect(resp.status).toEqual(302);
-    expect(mockTransactionService).toHaveBeenCalledTimes(1);
-    expect(mockOverseasEntity).toHaveBeenCalledTimes(1);
-    expect(resp.text).toEqual(`${FOUND_REDIRECT_TO} ${CONFIRMATION_URL}`);
-  });
-
-  test(`redirect to ${PAYMENT_LINK_JOURNEY}, the first Payment web journey page`, async () => {
-    mockIsActiveFeature.mockReturnValueOnce( false ); // For Save and Resume
-    mockPaymentsSession.mockReturnValueOnce(PAYMENT_LINK_JOURNEY);
-    const resp = await request(app).post(CHECK_YOUR_ANSWERS_URL);
-
-    expect(resp.status).toEqual(302);
-    expect(mockTransactionService).toHaveBeenCalledTimes(1);
-    expect(mockOverseasEntity).toHaveBeenCalledTimes(1);
-    expect(resp.text).toEqual(`${FOUND_REDIRECT_TO} ${PAYMENT_LINK_JOURNEY}`);
-  });
-
   test(`catch error when post data from ${CHECK_YOUR_ANSWERS_PAGE} page`, async () => {
-    mockOverseasEntity.mockImplementation(() => {
+    mockGetApplicationData.mockImplementation(() => {
```

Comment:
the mockOverseasEntity has now been removed from test so needed another mock to throw the error